### PR TITLE
Add Option to Hide Start/Locate Buttons

### PR DESCRIPTION
**Pull request by rsegers**
This PR add the option to hide the Start and/or Locate buttons on the toolbar. The Locate button is not supported by some vacuums and therefore has no added value. The Start button may not function well for homes with advanced layouts or schedules, and therefore use a custom action to start for example a daily routine, or a certain room.

The toolbar actions will align to the center when both the Start and Locate button are hidden.

The following two config items are added:

| Name                |   Type    | Default      | Description                                                             |
| ------------------- | :-------: | ------------ | ----------------------------------------------------------------------- |
| `show_start_button`  | `boolean` | `true`       | Show start button on toolbar.                                           |
| `show_locate_button` | `boolean` | `true`       | Show locate button on toolbar.                                          |

The config is also added to the visual editor.

idle
docked no buttons
docked start
config

Addresses issue #194

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,19 +100,21 @@ actions:
...
+| `entity`            | `string`  | **Required** | An entity_id within the `vacuum` domain.                                |
+| `map`               | `string`  | Optional     | An entity_id within the `camera` domain, for streaming live vacuum map. |
+| `map_refresh`       | `integer` | `5`          | Update interval for map camera in seconds                               |
+| `image`             | `string`  | `default`    | Path to image of your vacuum cleaner. Better to have `png` or `svg`.    |
+| `show_name`         | `boolean` | `true`       | Show friendly name of the vacuum.                                       |
+| `show_status`       | `boolean` | `true`       | Show status of the vacuum.                                              |
+| `show_toolbar`      | `boolean` | `true`       | Show toolbar with actions.                                              |
+| `compact_view`      | `boolean` | `false`      | Compact view without image.                                             |
+| `show_startbutton`  | `boolean` | `true`       | Show start button on toolbar.                                           |
```
Comment (denysdovhan) [inline README.md:114]: Let's make it `show_start_button` and `show_locate_button`

File: src/vacuum-card.js [hunk 2]
```diff
@@ -396,24 +412,32 @@ class VacuumCard extends LitElement {
           </ha-icon-button>
         `;
 
+        const startButton = html`
+          <ha-icon-button
```
Comment (denysdovhan) [inline src/vacuum-card.js:416]: This is deprecated in the latest HA. Needs to be updated.

**Review by denysdovhan — CHANGES_REQUESTED** [review]

**Comment by rsegers** [comment]
Thanks for the review! The PR had been updated

**Comment by dark-spider** [comment]
Hello. When will the card be updated with this feature? Thank you

**Comment by denysdovhan** [comment]
FYI: Russia have a huge military buildup around the borders of my country. It’s getting hot, countries are evacuating their embassies and flights are getting canceled.

I’m busy with packing my stuff to move out from Kyiv for some time in order to keep me and my family safe from possible war actions.

Sorry I’m not going to work it this projects in the nearest weeks. Thanks for understanding 🙏🏻

Please, spread #StandWithUkraine and try to make your government representatives know that Ukraine needs help.

**Comment by emufan** [comment]
Would be great to have this for any button and not for the listed two. As is not merged for now, perhaps this can be improved?

E.G. I want to hide all, but show my shortcut ones, e.g. to set another icon than the standard ones.

**Comment by 17hoehbr** [comment]
I would also love an option to hide the stop button as it doesn't work on my vacuum.

**Comment by yummypho** [comment]
I have a Wyze vacuum, and the Locate doesn't seem to work.  It would be great to get this PR accepted so we can hide feature not relevant.   Thx!

**Pull request closed without merging**